.buildconfig.yml

```diff
@@ -176,6 +176,10 @@ projects:
     path: components/browser/menu
     description: 'A customizable menu for browsers.'
     publish: true
+  browser-menu2:
```

Comment:
I'm split on this name.  Part of me thinks something like `immutable-browser-menu` would be better, part of me prefers that this one clearly communicates which module should be preferred.  I think it's probably best left as is, but I'm on the fence.

Reply:
I'm going to stick with this name, since its already been merged in and agreed upon. I'm not a big number fan either which is why I originally built this in the same component.